American Whitewater/ViewControllers/Reaches/RunMapViewController.swift

```diff
@@ -99,7 +96,10 @@ class RunMapViewController: UIViewController, MKMapViewDelegate, CLLocationManag
             }
             
             // set the bounding region to the put in and take out
-            mapView.showAnnotations(mapView.annotations, animated: false)
+            let annotations = mapView.annotations.filter({ annotation in
+                !(annotation is MKUserLocation)
+            })
+            mapView.showAnnotations(annotations, animated: false)
```

Comment:
Not really a review more of a comment... aren't locations for a trace along the run available? Would be cool to show that too.

Reply:
Yeah they should be, that would be awesome.